tests/metrics/test_regression.py

```diff
@@ -64,18 +65,6 @@ def test_mse_squared(squared):
     assert abs(result - expected) < 1e-5
 
 
-def test_mean_squared_log_error():
```

Comment:
adding `mean_squared_log_error()` to the `metric_pairs` fixture means that this standalone test for it can be removed. Similarity between this and the `scikit-learn` equivalent is covered by `test_ok()` in this same file.